src/crowbar.ml

```diff
@@ -407,16 +407,16 @@ let run_all_tests tests =
      (* AFL mode *)
      let verbose = (Array.length Sys.argv = 3) in
      AflPersistent.run (fun () ->
-         let chan = open_in file in
-         let state = { chan = Chan chan; buf = Bytes.make 256 '0';
+         let fd = Unix.openfile file [] 0o000 in
```

Comment:
`[]` is wrong here, it should be `[Unix.O_RDONLY]`.

(I was curious why this seemed to work regardless: turns out Linux `#define`s O_RDONLY to `0`, so it happens to work. This isn't guaranteed by POSIX, and you're [supposed to pass one of `O_RDONLY`, `O_RDWR` or `O_WRONLY`](http://pubs.opengroup.org/onlinepubs/009695399/functions/open.html))